worker/earn.js

```diff
@@ -182,7 +182,10 @@ function earnStmts (data, { models }) {
   return [
     models.earn.create({ data }),
     models.user.update({
-      where: { id: userId },
+      where: {
+        id: userId,
+        deletedAt: null // Only update if user is not deleted
```

Comment:
mhhh, need to think more if this is needed (how can a deleted user earn rewards?) and what the implications are if this is indeed needed (do we need to do something similar in other places, too?)

Reply:
If the content is still zapable or if the account is removed after some actions were done by the user before the rewards windows closes. Then there's referrals, etc.